tests/base.py

```diff
@@ -180,11 +180,18 @@ def is_eth_and_has_roce_hw_bug(self):
             has_roce_hw_bug(vendor_id, vendor_pid)
 
     @staticmethod
-    def get_net_name(dev):
-        out = subprocess.check_output(['ls',
-                                       '/sys/class/infiniband/{}/device/net/'
-                                      .format(dev)])
-        return out.decode().split('\n')[0]
+    def get_net_name(dev, port=None):
+        if port is not None:
+            out = subprocess.check_output(['rdma', 'link',  'show', '-j'])
+            loaded_json = json.loads(out.decode())
+            for row in loaded_json:
+                if row['ifname'] == dev and row['port'] == port:
+                    net_name = row['netdev']
```

Comment:
This breaks all pyverbs tests for EFA provider.

Reply:
Why? Can you please provide a fix to this get_net_name() before release of rdma-core?

Reply:
Sure. Already have a fix, will send it shortly.

Reply:
Please see [this PR](https://github.com/linux-rdma/rdma-core/pull/1328).